README.md

```diff
@@ -44,6 +44,33 @@ The Oracle WebLogic Server Deploy Tooling is designed to support a wide range of
 - The attribute data types and how to get/set their values (which isn't as easy as it might sound)
 - The differences between WLST online and WLST offline for working with folders and attributes
 
+### Supported WebLogic Server Releases
+The chart below specifies the supported WebLogic Server release, along with the JDK version(s) that must be used to run the WDT tool. You must set the JAVA_HOME environment variable to specify a version of a JDK different than the system default version. 
```

Comment:
The chart below -> The following table
supported WebLogic Server release -> supported WebLogic Server releases
JDK version(s) -> JDK versions
JAVA_HOME -> `JAVA_HOME` (code font)
specify a version of a JDK different than -> specify a different JDK version than